pkg/acs/lib/acs2d/doflash.c

```diff
@@ -74,8 +74,7 @@ int doFlash (ACSInfo *acs2d, SingleGroup *x, float *meanflash) {
 
     /* Check to see whether we need to do any processing at all */
     if (acs2d->flashdur <= 0.) {
-        sprintf(MsgText,
-            "Post-flash exposure was 0 seconds. FLSHCORR not performed.");
+        snprintf(MsgText, sizeof(MsgText), "Post-flash exposure was 0 seconds. FLSHCORR not performed.");
         trlwarn(MsgText);
```

Comment:
Why not shorten it like you did with `trlmessage` below? Same comment applies to all similar occurences. Also I like the simplification!
```suggestion
        trlwarn("Post-flash exposure was 0 seconds. FLSHCORR not performed.");
```

Reply:
I missed it, that's all. Thanks!